yt_idv/scene_components/base_component.py

```diff
@@ -17,14 +17,16 @@
 )
 
 _cmaps = ["arbre", "viridis", "magma", "doom"]
-
+_buffers = ["frame", "depth"]
 
 class SceneComponent(traitlets.HasTraits):
     data = traitlets.Instance(SceneData)
     base_quad = traitlets.Instance(SceneData)
     name = "undefined"
     priority = traitlets.CInt(0)
     visible = traitlets.Bool(True)
+    use_db = traitlets.Bool(False)
```

Comment:
```suggestion
    use_db = traitlets.Bool(False)  # use depth buffer
```

just adding that comment because my brain saw "db" and went straight to "database" instead of "depth buffer".